extensions/tasks-parent/tasks-ui/src/taskclient/task-types.ts

```diff
@@ -40,15 +36,17 @@ export interface TaskTransaction {
   commands: TaskCommand[];
 }
 
-export interface TaskExtension {
+export interface TaskItem {
```

Comment:
In here, we don't need this TaskItem interface at all.  You can add taskId directly to TaskExtension and TaskComment.